Event Page Redesign

ShivaniVora opened the pull request:
Event Page UI Cleanup

Known Issues
- No pagination, some commented code using Flowbite component
- Spacing issue when day from calendar with no events is clicked on (functional if the date clicked has events)
- Private event tag is not flowbite component

ShivaniVora committed:
event page restyling

colin4554 commented on src/screens/Events/EventManager.jsx:
```diff
@@ -347,55 +343,74 @@ const EventManager = ({ user, role, isHomePage }) => {
             <Styled.TablePadding></Styled.TablePadding>
           )}
           {events.length === 0 ? (
-            <Styled.Events>No Events Scheduled on This Date</Styled.Events>
```
Please just keep this as text, instead of a non themable image.

colin4554 commented on src/components/EventCard.jsx:
```diff
@@ -59,12 +59,25 @@ const EventCard = (props) => {
+          )}
           <div className="flex-column flex text-xl">
-            <Text text={props.event.title} type="subheading" />
+            <div className="flex">
```
Is there a reason you didn't use the flowbite component? https://flowbite-react.com/badges

colin4554 commented on src/screens/Events/EventsList.jsx:
```diff
@@ -140,18 +142,63 @@ const EventsList = ({
+          <Pagination
+            currentPage={pageNum}
+            layout="table"
+            onPageChange={onPageChange}
```
Look at the existing pagination implementations for ideas. The idea is that we slice the list of events based on what page is selected

ShivaniVora committed:
added pagination and badge component

ShivaniVora commented:
Added pagination and changed private tag to flowbite badge component, removed image on days with no events
- spacing on days with no events is not correct
- coloring of pagination is not themed
- coloring of flowbite badge is blue and static - not themed

ShivaniVora committed:
Merge branch 'dev' into event-page

ShivaniVora committed:
prettier

ShivaniVora committed:
Merge branch 'dev' into event-page

ShivaniVora committed:
fixed merge w dev

colin4554 committed:
Merge branch 'dev' into event-page

colin4554 committed:
update dropdown

colin4554 committed:
Merge branch 'dev' into event-page

colin4554 committed:
small cleanups

colin4554 committed:
remove old styled components
